src/build.d

```diff
@@ -987,6 +996,38 @@ version(Windows)
         // Could not find or obtain vswhere.exe
         throw new Exception("Could not obtain vswhere.exe. Consider downloading it from https://github.com/Microsoft/vswhere and placing it in your PATH");
     }
+
+    /*
+    Gets the absolute path to the host's MSVC bin directory
+
+    Params:
+        model   = a string describing the host's model, "64" or "32"
+        vswhere = a string that is the path to the vswhere executable
+    Returns: a string that is the absolute path to the host's MSVC bin directory
+    */
+    auto getHostMSVCBinDir(string model, string vswhere)
+    {
+        // See https://github.com/Microsoft/vswhere/wiki/Find-VC
+
+        const vsInstallPath = [vswhere, "-latest", "-products", "*", "-requires",
+            "Microsoft.VisualStudio.Component.VC.Tools.x86.x64", "-property", "installationPath"].execute.output
+            .replace("\r\n", "");
+
+        if (!vsInstallPath.exists)
+            throw new Exception("Could not locate the Visual Studio installation directory");
+
+        const vcVersionFile = vsInstallPath.buildPath("VC", "Auxiliary", "Build", "Microsoft.VCToolsVersion.default.txt");
+        if (!vcVersionFile.exists)
+            throw new Exception(`Could not locate the Visual C++ version file "%s"`.format(vcVersionFile));
+
+        const vcVersion = ["cmd", "/C", "type", `%s`.format(vcVersionFile)].execute.output.replace("\r\n", "");
```

Comment:
Why execute a rather expensive `cmd /c type <versionFile>` instead of just `std.file.read(versionFile)`?

Reply:
Because sometimes I'm a fool.  Fixed.